src/seer/seer.py

```diff
@@ -66,11 +66,18 @@ def breakpoint_trends_endpoint():
 
         sort_function = data.get('sort', "")
 
+        lower_limit_trend_percentage = data.get('trend_percentage()', 0.05)
+
+        if lower_limit_trend_percentage[0] == ">":
+            lower_limit_trend_percentage = lower_limit_trend_percentage[1:]
+
+        lower_limit_trend_percentage = float(lower_limit_trend_percentage)
+
         with sentry_sdk.start_span(
                 op="cusum.detection", description="Get the breakpoint and t-value for every transaction"
         ) as span:
 
-            trend_percentage_list = find_trends(txns_data, sort_function, zerofilled)
+            trend_percentage_list = find_trends(txns_data, sort_function, zerofilled, lower_limit_trend_percentage)
```

Comment:
You could do something like this to avoid having to change the function signature at all. Though considering this is the only caller, and you have the default value in place, I don't have strong opinions here.

```suggestion
            trend_percentage_list = find_trends(txns_data, sort_function, zerofilled, trend_perc=lower_limit_trend_percentage)
```